### PR TITLE
Fix bug where eslint configuration could not resolve tsconfig in VSCode

### PR DESCRIPTION
**Pull request by juanscr**
This adds a configuration to the workspace vscode settings to make vscode recognize the proper eslint configuration file and the relative working directory for each sub-package in the monorepo.

### Main changes in the PR:

1. Add working directories settings through workspace settings in vscode

## Validation

I opened the files locally and created apparent error lines that eslint should point out and it work flawlessly. As this change is done through the vscode setting, this does not impact any other outside script other than the editor experience.

### Unit Tests added:

Not needed as it related to developer experience instead of a new feature or bug.

### End-to-end tests added:

No

### Change file added:

No

### Screenshots:

> Remove this section if n/a

| Before     | After      |
| ---------- | ---------- |
| image |  image |

Notice the error in the editor disappeared.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,6 +1,13 @@
 {
   "cSpell.words": ["adal", "ipados", "teamspace", "uninitialize", "xvfb"],
   "editor.defaultFormatter": "esbenp.prettier-vscode",
+  "eslint.workingDirectories": [
+    "./packages/teams-js/",
+    "./apps/teams-perf-test-app",
+    "./apps/ssr-test-app/",
+    "./apps/teams-test-app/",
+    "./apps/typed-dependency-tester/"
```
Comment (AE-MS) [inline .vscode/settings.json:9]: Can you alphabetize these? Thanks!

**Comment by AE-MS** [comment]
Thanks for fixing this!

File: .vscode/settings.json (hunk 1 above)
Comment (juanscr) [reply .vscode/settings.json:9]: Yes! Give me sec

**Review by AE-MS — CHANGES_REQUESTED** [review]
:clock1:

**Review by TrevorJoelHarris — APPROVED** [review]

**Pull request merged**